app/experimenter/nimbus-ui/src/components/Summary/index.test.tsx

```diff
@@ -10,31 +10,72 @@ import {
   NimbusExperimentPublishStatus,
   NimbusExperimentStatus,
 } from "../../types/globalTypes";
-import { createMutationMock, reviewRequestedBaseProps, Subject } from "./mocks";
+import { createMutationMock, Subject } from "./mocks";
 
 describe("Summary", () => {
   it("renders expected components", () => {
     render(<Subject />);
     expect(screen.getByTestId("summary-timeline")).toBeInTheDocument();
     expect(screen.queryByTestId("experiment-end")).not.toBeInTheDocument();
-    expect(screen.getByTestId("table-summary")).toBeInTheDocument();
+    expect(
+      screen.queryByTestId("link-monitoring-dashboard"),
+    ).not.toBeInTheDocument();
+    expect(screen.getByTestId("table-overview")).toBeInTheDocument();
     expect(screen.getByTestId("table-audience")).toBeInTheDocument();
     expect(screen.queryAllByTestId("table-branch")).toHaveLength(2);
     expect(screen.getByTestId("branches-section-title")).toHaveTextContent(
       "Branches",
     );
   });
 
-  it("renders end experiment badge if end is requested", async () => {
+  it("renders monitoring dashboard URL if experiment has been launched", async () => {
     render(
       <Subject
         props={{
           status: NimbusExperimentStatus.LIVE,
-          statusNext: NimbusExperimentStatus.COMPLETE,
         }}
       />,
     );
-    await screen.findByTestId("pill-end-requested");
+
+    await screen.findByTestId("link-monitoring-dashboard");
+  });
+
+  it("does not render monitoring dashboard URL if experiment has not been launched", async () => {
```

Comment:
Nit for some of these tests (100% not blocker tho, feel free to ignore): they don't need to be async functions if there are no async calls